Python_PostBuild/Program.cs

```diff
@@ -14,29 +14,29 @@ static void Main(string[] args)
             if (args.Contains("--force"))
                 force = true;
 
-            // Install python
+            // Install python	
```

Comment:
Is there a reason you haven't used the `Compute.InstallPythonToolkit` method here?

Reply:
> I have removed the postbuild from the SLN file but kept it within the repo for now until we're happy.

Mainly I am thinking we will remove the Python_PostBuild project if it adds nothing to the use case.

That said, I am happy to convert it to use the `Compute.Install` method to be up to date with current working until we remove it :smile: